v20.1/cockroachcloud-cluster-management.md

```diff
@@ -32,11 +28,26 @@ For each node, the page displays the node's `Name`, nested under its region.
 
 From the **Cluster Overview** page, you can connect to your cluster. For more information, see [Connect to Your CockroachCloud Cluster](cockroachcloud-connect-to-your-cluster.html).
 
-## Manage cluster configuration
+## Add or remove nodes from a cluster
+
+You can add or remove nodes from your cluster through the Console.
+
+{{site.data.alerts.callout_info}}
+At this time, you cannot use the Console to scale up a single-node cluster or scale down to a single-node cluster. For single-node cluster changes, contact [Support](https://support.cockroachlabs.com).
+{{site.data.alerts.end}}
+
+### Considerations
+
+- We recommend you add or remove nodes from a cluster when the cluster isn't experiencing heavy traffic. Adding or removing nodes incurs a non-trivial amount of load on the cluster. Changing the cluster configuration during times of heavy traffic can result in degraded application performance or longer times for node modifications. 
```

Comment:
I might reverse the order of these 2 sentences so it's "problem -> solution"

Reply:
I went with the "state the action and then the reason" structure, but I don't mind the problem -> solution structure. Done!